src/rule-transform/rule-content/get-examples-content.ts

```diff
@@ -71,3 +69,57 @@ function getExternalLink(
     `href="${href}">Open in a new tab</a> `
   );
 }
+
+function getAssetsString(assets: TestAssets = {}): string {
+  const sorted = Object.entries(assets).sort((a, b) => comparer(a[0], b[0]));
+
+  if (sorted.length === 0) {
+    return "";
+  }
+
+  const plural = sorted.length === 1 ? ["This", " is"] : ["These", "s are"];
+  const hasJS =
+    sorted.find(([filename]) => filename.endsWith(".js")) !== undefined;
+  const js = hasJS ? "Javascript " : "";
+  const hasCSS =
+    sorted.find(([filename]) => filename.endsWith(".css")) !== undefined;
+  const css = hasCSS ? "CSS " : "";
+  const both = hasJS && hasCSS ? "and " : "";
+
+  const header = `${plural[0]} ${js}${both}${css}file${plural[1]} used in several examples:`;
+
+  const assetsBase = "/WAI/content-assets/wcag-act-rules";
+  const blocks: string[] = [];
+
+  for (const [filename, content] of sorted) {
+    const truePath = filename.replace(
+      /\/test-assets\//g,
+      `${assetsBase}/test-assets/`
+    );
+
+    blocks.push(
+      joinStrings(`File [\`${filename}\`](${truePath}):`, [
+        "```" + (filename.endsWith(".js") ? "javascript" : "css"),
+        content + (content.endsWith("\n") ? "" : "\n") + "```",
+      ])
+    );
+  }
+
+  return joinStrings(
+    ["<details>", "<summary>", header, "</summary>"],
```

Comment:
1. Add a class so this can be styled
2. markdown=block so the markdown parser processes the content as markdown
3. put the summary elm on one line (personal preference, can do without that if you don't like it)

```suggestion
    ["<details class="act-inline-assets" markdown="block">", `<summary>${header}</summary>`],
```